stdlib/logging/__init__.pyi

```diff
@@ -59,7 +60,7 @@ class Logger(Filterer):
     def setLevel(self, level: _Level) -> None: ...
     def isEnabledFor(self, level: int) -> bool: ...
     def getEffectiveLevel(self) -> int: ...
-    def getChild(self, suffix: str) -> Logger: ...
+    def getChild(self: _LoggerType, suffix: str) -> _LoggerType: ...
```

Comment:
It would be better to use `_typeshed.Self` for this:

```suggestion
    def getChild(self: Self, suffix: str) -> Self: ...
```

You will need to add `from _typeshed import Self`.

Reply:
So the `bound=` is generally not necessary for this type of operation?

Reply:
That's right. `self: T` creates an implicit bound on `T`.

Reply:
Maybe I should clarify this a bit more. This is fine:

```
T = TypeVar("T")

class Foo:
    def do_something(self: T) -> T: ...  # T has implicitly bound=Foo

def bar(thing: T) -> T: ...   # no bound on T
```